.eslintrc.json

```diff
@@ -4,15 +4,27 @@
     "next",
     "next/core-web-vitals",
     "prettier",
-    "plugin:storybook/recommended"
+    "plugin:storybook/recommended",
+    "plugin:@typescript-eslint/recommended"
```

Comment:
This extend should not be here. It should be below line 20.

Reply:
Okay, got it!
Moved it there.